pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -0,0 +1,110 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Import Project="$(MSBuildProjectName).props" />
+
+  <PropertyGroup>
+    <IsLineupPackage Condition="'$(PackageTargetRuntime)' == ''">true</IsLineupPackage>
+    <PreventImplementationReference Condition="'$(PackageTargetRuntime)' != ''">true</PreventImplementationReference>     
+    <TargetFrameworkName>uap</TargetFrameworkName>
+    <TargetFrameworkVersion>10.1</TargetFrameworkVersion>
+    <TargetFramework>$(TargetFrameworkName)$(TargetFrameworkVersion)</TargetFramework>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+
+    <RefBinDir>$(UAPPackageRefPath)</RefBinDir>
+    <LibBinDir>$(UAPPackageRuntimePath)</LibBinDir>
+
+    <NETStandardLibraryPackage>NETStandard.Library2</NETStandardLibraryPackage>
+    <NETStandardLibraryPackageVersion>2.0.0-beta-24709-0</NETStandardLibraryPackageVersion>
+    <NETStandardVersion>2.0</NETStandardVersion>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <ProjectReference Include="@(Project)" />
+
+    <!-- Include refs -->
+    <RefFile Include="$(RefBinDir)/*.*">
+      <TargetPath>ref/$(TargetFramework)</TargetPath>
+    </RefFile>
+    <File Include="@(RefFile)" />
+    <!-- force a missing file if ref build is absent -->
+    <File Include="$(RefBinDir)/MISSING_REF_BUILD" Condition="'@(RefFile)' == ''" />
+
+    <FilePackageDependency Include="Microsoft.NETCore.Platforms">
+      <Version>$(PlatformPackageVersion)</Version>
+    </FilePackageDependency>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+    <!-- Include lib -->
+    <LibFile Include="$(LibBinDir)/*.*">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/lib/$(TargetFramework)</TargetPath>
+    </LibFile>
+    <File Include="@(LibFile)" />
+    <!-- force a missing file if lib build is absent -->
+    <File Include="$(LibBinDir)/MISSING_LIB_BUILD" Condition="'@(LibFile)' == ''" />
+
+    <!-- Include native -->
+    <ExcludeNative Include="$(NativeBinDir)/*.lib" />
+    <NativeFile Include="$(NativeBinDir)/*.*" Exclude="@(ExcludeNative)">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+    </NativeFile>
+    <File Include="@(NativeFile)" />
+    <!-- force a missing file if native build is absent -->
+    <File Include="$(NativeBinDir)/MISSING_NATIVE_BUILD" Condition="'@(NativeFile)' == ''" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <IgnoredReference Include="System.Private.CoreLib" />
+    <IgnoredReference Include="Windows" />
+    <IgnoredReference Include="Microsoft.Win32.Registry" /> <!-- Ignore for now since it's being exclude from closure bellow -->
```

Comment:
Can you please make sure there are bugs tracking each of these issues and add a reference to the issue in the comment?